### DISCUSSION TITLE
Welcome to comments Discussions!

### DISCUSSION THREAD

**Discussion by brycewray** [brycewray/comments#1] [Announcements]
This gets us started with GH Discussions so [giscus](https://giscus.app) will work with it from the appropriate site repo.

**Edit**: I later changed the setup so that they in fact work from this `comments` repo, rather than a site-specific repo, which protects them from my tendency to migrate my site to different repos.

**Comment by leafstrat** [comment]
I've appreciated your blog posts on your experience with giscus, thanks for sharing. 
Added ya to the ole RSS feed :)

**Reply by brycewray** [reply]
@leafstrat Thank you! Much appreciated.
